pkg/operator/ceph/cluster/mon/node.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
```suggestion
Copyright 2019 The Rook Authors. All rights reserved.
```

Reply:
@travisn This would be a good time to ask about when/if to update these copyright comments. The file itself may be new, but the contents are largely from a file with the 2016 header. Or should we update the date any time a file is modified?

Reply:
For existing or moved files i wouldn't worry about updating the copyright